code/modules/antagonists/cult/cult_structures.dm

```diff
@@ -79,7 +79,7 @@
 
 /obj/structure/destructible/cult/talisman
 	name = "altar"
-	desc = "A bloodstained altar dedicated to Nar'Sie."
+	desc = "A bloodstained altar dedicated to Nar'Sie. A blood cultist can use it to create eldritch whetstones, construct shells, and flasks of unholy water."
```

Comment:
Have this in an examine message for cultists only.

Reply:
how would I do that?

Reply:
add a new var on the parent item, var/cultist_examine_message
then do 
/obj/structure/destructible/cult/examine(mob/user)
. = ..()
if(iscultist(user)
. += to_chat(user, "<span class='cult'>[cultist_examine_message]</span>"

Reply:
it'll need a space before the examine message, right?

Reply:
No, it'll be on a new line automatically.